vision_evaluation/evaluators.py

```diff
@@ -417,13 +483,21 @@ def _calculate_average_precision(is_correct, probabilities, true_num, average='m
         return sm.average_precision_score(is_correct, probabilities, average=average) * recall
 
     def get_report(self, **kwargs):
+        """
+        Args:
+            labels : If report_per_class, pass the list of class names the same order as the ground_truths label index
+        """
         average = kwargs['average'] if 'average' in kwargs else 'macro'
         for class_index in self.is_correct:
             ap = MeanAveragePrecisionEvaluatorForSingleIOU._calculate_average_precision(self.is_correct[class_index], self.probabilities[class_index], self.true_num[class_index], average)
             self.aps[class_index] = ap
 
         mean_ap = float(statistics.mean([self.aps[x] for x in self.aps])) if self.aps else 0.0
-        return {f'mAP_{int(self.iou * 100)}': mean_ap}
+        report = {f'mAP_{int(self.iou * 100)}': mean_ap}
```

Comment:
f'mAP_{int(self.iou * 100)}' appeared twice. assign it to a variable. e.g. key_name = f'mAP_{...}'